hack/.golint_failures

```diff
@@ -646,4 +645,5 @@ test/e2e_node
 test/e2e_node/remote
 test/e2e_node/runner/remote
 test/e2e_node/services
+test/integration/apiserver/admissionwebhook
```

Comment:
This shouldn't be required. Master is green without it and verify is failing saying it isn't necessary